cmd/validate-krew-manifest/main.go

```diff
@@ -40,29 +40,29 @@ var flManifest string
 
 func init() {
 	flag.StringVar(&flManifest, "manifest", "", "path to plugin manifest file")
-	if err := flag.Set("logtostderr", "true"); err != nil {
-		fmt.Printf("can't set log to stderr %+v", err)
-		os.Exit(1)
-	}
 }
 
 func main() {
 	// TODO(ahmetb) iterate over glog flags and hide them (not sure if possible without using pflag)
+	klog.InitFlags(nil)
```

Comment:
I looks a bit inconsistent that for the main command the setup is done in `init` whereas for validate-krew-manifest it's done in `main()`.